credoai/evaluators/utils/utils.py

```diff
@@ -15,3 +19,34 @@ def all_subclasses(cls):
     return set(cls.__subclasses__()).union(
         [s for c in cls.__subclasses__() for s in all_subclasses(c)]
     )
+
+
+def list_evaluators_exhaustive():
+    """List all defined evaluation modules"""
+    evaluator_path = os.path.dirname(os.path.dirname(__file__))
```

Comment:
Will we ever have a case where evaluators are organized into further subfolders?

If so, this would need to be changed (heads up, but don't think it's a present problem to worry about)

Reply:
I think this actually goes into subfolders, but yeah, this is based on our current organization.